docs/ingestion/compaction.md

```diff
@@ -62,16 +71,12 @@ Unless you modify the query granularity in the [granularity spec](#compaction-gr
 If you configure query granularity in compaction to go from a finer granularity like month to a coarser query granularity like year, then Druid overshadows the original segment with coarser granularity. Because the new segments have a coarser granularity, running a kill task to remove the overshadowed segments for those intervals will cause you to permanently lose the finer granularity data.
 
 ### Dimension handling
-Apache Druid supports schema changes. Therefore, dimensions can be different across segments even if they are a part of the same data source. See [Different schemas among segments](../design/segments.md#different-schemas-among-segments). If the input segments have different dimensions, the resulting compacted segment includes all dimensions of the input segments. 
+Apache Druid supports schema changes. Therefore, dimensions can be different across segments even if they are a part of the same data source. See [Different schemas among segments](../design/segments.md#different-schemas-among-segments). If the input segments have different dimensions, the resulting compacted segment include all dimensions of the input segments. 
 
 Even when the input segments have the same set of dimensions, the dimension order or the data type of dimensions can be different. The dimensions of recent segments precede that of old segments in terms of data types and the ordering because more recent segments are more likely to have the preferred order and data types.
 
 If you want to use your own ordering and types, you can specify a custom `dimensionsSpec` in the compaction task spec.
 
-### Rollup
```

Comment:
Why is this removed?

Reply:
This is still true. What I meant to say is that compaction cannot change the `rollup` setting but the `rollup` setting on the original segments is important and determines how compaction works